crypto/src/test/java/tech/pegasys/pantheon/crypto/Blake2bfMessageDigestTest.java

```diff
@@ -27,21 +27,21 @@
   private Blake2bfMessageDigest messageDigest;
 
   // output when input is all 0
-  private byte[] blake2bfAllZero =
+  private static final byte[] BLAKE2F_ALL_ZERO =
```

Comment:
Where are these test cases coming from?  Is there a spec we can link to?  Would the tests be clearer if kept these values as hexary strings?

Reply:
These tests are not as useful as the precompiled test vectors.  Those have more coverage.  As far as I can tell they were calculated from the blake RFC.